proto/tendermint/liquidity/v1beta1/tx.proto

```diff
@@ -159,14 +163,14 @@ message MsgSwapWithinBatch {
       format: "uint64"
     }];
 
-  // id of swap type, only 1 is allowed on this version, Must match the value in the pool.
+  // id of swap type, must match the value in the pool. Only `swap_type_id` 1 is supported.
```

Comment:
same question as above, if `1` is the only supported value, why do we shoe `3` on line 167? just asking to learn